Description: add missing linkage on libgslcblas to libgsl.so
 libgsl has missing symbols because it is not linked against libgslcblas.
 Fix it so the library is properly stand-alone.
Author: Steve Langasek <steve.langasek@ubuntu.com>
Last-Modified: 2018-08-23

Index: gsl-2.5+dfsg/Makefile.am
===================================================================
--- gsl-2.5+dfsg.orig/Makefile.am
+++ gsl-2.5+dfsg/Makefile.am
@@ -19,7 +19,7 @@
 
 lib_LTLIBRARIES = libgsl.la
 libgsl_la_SOURCES = version.c
-libgsl_la_LIBADD = $(GSL_LIBADD) $(SUBLIBS)
+libgsl_la_LIBADD = $(GSL_LIBADD) $(SUBLIBS) cblas/libgslcblas.la
 libgsl_la_LDFLAGS = $(GSL_LDFLAGS) -version-info $(GSL_LT_VERSION)
 noinst_HEADERS = templates_on.h templates_off.h build.h

